AirCasting/SessionViews/ShareSession/ShareLocationlessSessionViewModel.swift

```diff
@@ -54,6 +54,14 @@ class ShareLocationlessSessionViewModel: ObservableObject {
         exitRoute()
     }
     
+    func getSharePage() -> ActivityViewController? {
+        if file != nil {
```

Comment:
I know it is how I proposed it, but I noticed now that using guard would probably be more swifty version (at least that's what Paweł usually says 😄 )

Reply:
Will do then 😎